components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -343,19 +343,34 @@ fun onShowLogin(e: Event) {
 }
 
 fun onLogin(e: Event) {
-    Auth.loginTime.stop(e.target)
+    Auth.loginTime.stopAndSum(e.target)
     // ...
 }
 
 fun onLoginCancel(e: Event) {
-    Auth.loginTime.cancel(e.tart)
+    Auth.loginTime.cancel(e.target)
     // ...
 }
 ```
 
 The time reported in the telemetry ping will be the sum of all of these
 timespans recorded during the lifetime of the ping.
 
+If the above `start`/`stopAndSum`/`cancel` API isn't practical to use from your
```

Comment:
I would be very explicit here: "If the above `start`/`stopAndSum`/`cancel` API can't be used due to restrictions (e.g. using 3rd party library that is providing raw timespan values), you may use..."